src/SonarScanner.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -149,26 +150,44 @@
     <Error Text="SonarQube analysis is only supported with MSBuild 14 or later." />
   </Target>
 
+  <Target Name="SonarResolveReferencesMsBuild14"
```

Comment:
nitpick: I would add a comment above these two targets saying that `SonarResolvedReferences` are the references resolved by MSBuild (they're not resolved by `Sonar`, like the name might suggest), and that we prefix with `Sonar` to avoid name clashes

Reply:
IMO `Sonar` should be prefix for all our targets, property and item groups to avoid clash with user pipelines. We should not need this comment as it should be the other way around - anything non-sonar is expected to come from MsBuild.